### PR TITLE
Feature/backtrack status notifications

### PR DESCRIPTION
**Pull request by Baggerone**
Also,  
 - moved email subjects out of the sender functions and grouped together in the statusSenders map
 - moved create and pop functions for PostHistories to that file (out of posts.go)

### CODE REVIEW

File: application/templates/mail/request_from_accepted_to_committed.html [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<p>A requester on <a href="<%= uiURL %>"><%= appName %></a> isn't sure yet if they
+    want you to fulfill that request.</p>
+<h4><a href="<%= postURL %>"><%= postTitle %></a></h4>
+<p>The requester, <%= requesterNickname %>, isn't ready afterall to have you
```
Comment (Schparky) [inline application/templates/mail/request_from_accepted_to_committed.html]:
```suggestion
<p>The requester, <%= requesterNickname %>, isn't ready after all to have you
```

File: application/templates/mail/request_from_delivered_to_accepted.html [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<p>The person who initially claimed to have to fulfilled your request on
+    the <a href="<%= uiURL %>"><%= appName %></a> now says they haven't yet delivered it.</p>
+<h4><a href="<%= postURL %>"><%= postTitle %></a></h4>
+
+<p><strong><%= providerNickname %></strong> corrected themselves to say they haven't
+    delivered your request afterall.</p>
```
Comment (Schparky) [inline application/templates/mail/request_from_delivered_to_accepted.html]:
```suggestion
    delivered your request after all.</p>
```

File: application/templates/mail/request_from_delivered_to_committed.html [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<p>The person who initially claimed to have fulfilled your request on
+    the <a href="<%= uiURL %>"><%= appName %></a> now says they haven't yet delivered it.</p>
+<h4><a href="<%= postURL %>"><%= postTitle %></a></h4>
+
+<p><strong><%= providerNickname %></strong> corrected themselves to say they haven't
+    delivered your request afterall.</p>
```
Comment (Schparky) [inline application/templates/mail/request_from_delivered_to_committed.html]:
```suggestion
    delivered your request after all.</p>
```

File: application/templates/mail/request_not_received_afterall.html [hunk 4]
```diff
@@ -0,0 +1,7 @@
+<p>A requester on <a href="<%= uiURL %>"><%= appName %></a> said they haven't received the item afterall.</p>
```
Comment (Schparky) [inline application/templates/mail/request_not_received_afterall.html]:
```suggestion
<p>A requester on <a href="<%= uiURL %>"><%= appName %></a> said they haven't received the item after all.</p>
```

File: application/templates/mail/request_not_received_afterall.html [hunk 5]
```diff
@@ -0,0 +1,7 @@
...
+<h4><a href="<%= postURL %>"><%= postTitle %></a></h4>
+<p>The requester, <%= requesterNickname %>, initially said they received it from you but now have corrected that
+    to say they haven't received it afterall. You can chat with them <a href="<%= postURL %>">here</a>,
```
Comment (Schparky) [inline application/templates/mail/request_not_received_afterall.html]:
```suggestion
    to say they haven't received it after all. You can chat with them <a href="<%= postURL %>">here</a>,
```

File: application/templates/mail/request_from_accepted_to_committed.html (hunk 1 above)
Comment (Baggerone) [reply application/templates/mail/request_from_accepted_to_committed.html]: Oops - who would have thought it's two words. (-;

**Pull request merged**